src/libs/PolicyUtils.ts

```diff
@@ -526,12 +528,37 @@ function getDefaultApprover(policy: OnyxEntry<Policy>): string {
 }
 
 /**
- * Returns the accountID to whom the given employeeAccountID submits reports to in the given Policy.
+ * Returns the accountID to whom the given expenseReport submits reports to in the given Policy.
  */
-function getSubmitToAccountID(policy: OnyxEntry<Policy>, employeeAccountID: number): number {
+function getSubmitToAccountID(policy: OnyxEntry<Policy>, expenseReport: OnyxEntry<Report>): number {
```

Comment:
Just curious, is this called anywhere for `control` policies? If so, we shouldn't consider category / tag approvers, just the policy's `defaultApprover` (`approver` key)

Reply:
@nkdengineer my last question for this PR ^